main.tf

```diff
@@ -1,9 +1,9 @@
-resource "aci_rest_managed" "fvTenant" {
-  dn         = "uni/tn-${var.name}"
-  class_name = "fvTenant"
+resource "aci_rest_managed" "bgpBestPathCtrlPol" {
+  dn         = "uni/tn-${var.tenant}/bestpath-${var.name}"
+  class_name = "bgpBestPathCtrlPol"
   content = {
-    name      = var.name
-    nameAlias = var.alias
-    descr     = var.description
+    name  = var.name
+    descr = var.description
+    ctrl  = var.control_type == "multi-path-relax" ? "asPathMultipathRelax" : ""
```

Comment:
@danischm . The only values supported for the attribute `ctrl` are: `asPathMultipathRelax` and ``.
It would make more sense to use a flag here, however it will introduce breaking changes in Ansible. What is you opinion?